helm/values.yaml

```diff
@@ -0,0 +1,108 @@
+# Default values for indexd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: docker.osdc.io/indexd
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
```

Comment:
Do we need the service account?

Reply:
It is better to use service accounts to deploy services for security reasons. Applications deployed in kubernetes can technically access the kubernetes API and with service accounts you can limit what resources they have access to. If a service account is not provided it assumes the default service account from that namespace.

Reply:
I see. I had incorrectly assumed the pods would be anonymous without that. This makes sense.